### ISSUE TITLE
Building from source on ARM

### ISSUE THREAD

**Issue by eljoth** [improbable-eng/grpc-web#463]
**Versions of relevant software used**
Raspbian Stretch

**What happened**
I Followed the Building-from-source-Instructions

**What you expected to happen**
A binary in my bin-directory

**How to reproduce it (as minimally and precisely as possible)**:
Clone repo from git on ARM-Machine and follow rest of the instructions building it from source

**Full logs to relevant components**
```bash
go: finding google.golang.org/grpc/transport latest
build github.com/improbable-eng/grpc-web/go/grpcwebproxy: cannot load google.golang.org/grpc/transport: cannot find module providing package google.golang.org/grpc/transport
```

**Anything else we need to know**

I know this Issue is similar to #216 and that grpc/transport is now internal. But since mwitkow/grpc-proxy/pull/27 should have fixed it and building from source works on my x86-machine I am just confused how I couldn't work on my Raspberry Pi and I appreciate any help. 
It would be enough for me to just download the pre-build binary for ARM but I cannot find it either.

**Environment**:
- OS Raspbian Stretch
- Kernel 4.19.42-v7+

**Comment by johanbrandhorst** [comment]
This will be fixed by #395, but I am still confused as to why it's failing for you. I was able to manually build ARM binaries for the latest releases. It looks like you're using go modules to build (judging by the "finding" message), could you try again with Go modules disabled? This repo is still using dep (which should be part of the build instructions).

**Comment by eljoth** [comment]
Well, disabling go mod was the solution. Thanks!
